DigitalLearningSolutions.Data/Models/CustomPrompts/CentreRegistrationPromptWithAnswer.cs

```diff
@@ -2,8 +2,15 @@
 {
     public class CentreRegistrationPromptWithAnswer : CentreRegistrationPrompt
     {
-        public CentreRegistrationPromptWithAnswer(int customPromptNumber, string text, string? options, bool mandatory, string? answer)
-            : base(customPromptNumber, text, options, mandatory)
+        public CentreRegistrationPromptWithAnswer(
+            int customPromptNumber,
+            int promptId,
+            string text,
+            string? options,
+            bool mandatory,
+            string? answer
+        )
+            : base(customPromptNumber, promptId, text, options, mandatory)

```

Comment:
I would have expected the formatter to put this on the same line as the bracket?